Configuration/NodeTypes.FormElement.yaml

```diff
@@ -9,7 +9,7 @@
   constraints:
     nodeTypes:
       '*': false
-  label: "${q(node).property('identifier') || q(node).property('label') || ((node.nodeType.label || node.nodeTypeName.value) + ' (' + node.name + ')')}"
+  label: "${q(node).property('identifier') || q(node).property('label') || ((Neos.Node.getNodeType(node).label || node.nodeTypeName.value) + ' (' + node.name + ')')}"
```

Comment:
Do we have to load the whole node type to render its label or is there a dedicated helper?

Reply:
You're right, I can just get it form the node. Will fix :)